src/main/java/com/jwt/jwt/JwtConfig/JwtTokenUtil.java

```diff
@@ -0,0 +1,82 @@
+package com.jwt.jwt.JwtConfig;
+
+import com.jwt.jwt.Entity.UserData;
+import com.jwt.jwt.Repository.UserDataRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+
+import java.io.Serial;
+import java.io.Serializable;
+import java.util.*;
+import java.util.function.Function;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Component;
+
+@Component
+public class JwtTokenUtil implements Serializable {
```

Comment:
Actually the document I was following used serialization thats why I also did, but in our project don't need serialization.
Serializable interface required for converting object of that class into byte code so that we can store it into database, file or memory.

Reply:
Removed Serialization code